hugo/content/research/2022/_index.md

```diff
@@ -3,13 +3,36 @@ title: "DORA Research: 2022"
-# The data for survey questions can be found at data/survey_questions.json
-sections_to_display:
-    - 'Structural Equation Model'
-    - 'Questions'
 type: research_archives
+tab_order: "0"
+tab_title: "Overview"
+layout: single
 ---
+In 2022, DORA continued its investigation into software development and operations, and the capabilities which drive performance. This year's deep-dive investigations included Security, Reliability, and Culture. You can [download the 2022 report here]({{< relref "dora-report" >}}).
 
-# DORA's Research Program: 2022
-In 2022, DORA continued its investigation into software development and operations, and the capabilities which drive performance. This year's deep-dive investigations included Security, Reliability, and Culture. You can [download the 2022 report here]({{< relref "dora-report" >}}). Below are some of the analytical tools used in producing the report.
+### Key Findings
+
+{{< image src="dora-report/2022-dora-accelerate-state-of-devops-report.png" url="dora-report" float="right" width="14em" border="1px solid #eee" >}}
+
+#### The biggest predictor of an organization's application-development security practices is cultural, not technical
+High trust, low-blame cultures focused on performance were significantly more likely to adopt emerging security practices. We noticed that teams who focus on establishing these security practices demonstrate reduced developer burnout; teams with low levels of security practices have 1.4x greater odds of having high levels of burnout than teams with high levels of security practices. Further, teams with high levels of security practices are 1.6x more likely to have high levels of organizational performance than teams with low levels of security practices.
+
+#### The key variables that impact organizational performance tend to fall in the following categories:
+**Organizational and team culture**
+High-trust and low-blame cultures tend to have higher organizational performance. Similarly, organizations with teams that feel supported through funding and leadership sponsorships tend to have higher organizational performance. Team stability and positive perceptions about one’s team also tend to lead to higher levels of organizational performance. Lastly, the companies that offer flexible work arrangements are the companies that tend to see high levels of organizational performance. 
+
+**Reliability**
+Both the practices we associate with reliability engineering and the extent to which people report meeting their reliability expectations are powerful predictors of high levels of organizational performance.
+
+**Cloud**
+Teams continue to move workloads to the cloud and those that leverage all five capabilities of cloud see increases in software delivery and operational (SDO) performance and organizational performance. Multi-cloud adoption is also on the rise so that teams can leverage the unique capabilities of each provider.
```

Comment:
Remove `(SDO)`?

```suggestion
Teams continue to move workloads to the cloud and those that leverage all five capabilities of cloud see increases in software delivery and operational performance and organizational performance. Multi-cloud adoption is also on the rise so that teams can leverage the unique capabilities of each provider.
```